src/mnemonics/german.h

```diff
@@ -1678,11 +1678,11 @@ namespace Language
         "Zündung",
         "Zweck",
         "Zyklop"
-      }), 4)
-    {
+      }), 4)
+    {

```

Comment:
I saw those. I didn't change a thing though - do you know how I'd remove these from the commit?